package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "json-python-coverage",
+  "version": "1.0.0",
+  "description": "JavaScript tool for converting json coverage report to format used by Python coverage.",
+  "scripts": {
+    "test": "jest",
+    "check-js": "eslint --ext .js test src"
+  },
+  "keywords": [
+    "coverage-reporter",
+    "CLI"
+  ],
+  "repository": {
+    "type": "git",
+    "url": " https://github.com/CZ-NIC/json-python-coverage"
+  },
+  "bugs": {
+    "url" : "https://github.com/CZ-NIC/json-python-coverage/issues"
+  },
+  "license": "GPL-3.0-or-later",
+  "author": "Martin Strouhal <martin.strouhal@nic.cz>",
+  "preferGlobal": true,
```

Comment:
It seems that this option is DEPRECATED (see https://docs.npmjs.com/files/package.json#preferglobal).

Reply:
Fixed